mirar/pipelines/winter/generator.py

```diff
@@ -66,6 +69,15 @@
 logger = logging.getLogger(__name__)
 
 
+winter_cal_requirements = [
+    CalRequirement(
+        target_name="dark",
```

Comment:
Will this use only midnight darks?

Reply:
I think it should work - CalHunter uses the same image loading function as the raw images, so afternoon darks should be tagged as "test"

Reply:
Tested and made sure